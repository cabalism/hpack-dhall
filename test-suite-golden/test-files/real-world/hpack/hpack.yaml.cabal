cabal-version: 1.12

-- This file has been generated from hpack.yaml by hpack version 0.34.6.
--
-- see: https://github.com/sol/hpack

name:           hpack
version:        0.31.1
synopsis:       A modern format for Haskell packages
description:    See README at <https://github.com/sol/hpack#readme>
category:       Development
homepage:       https://github.com/sol/hpack#readme
bug-reports:    https://github.com/sol/hpack/issues
maintainer:     Simon Hengel <sol@typeful.net>
build-type:     Simple
extra-source-files:
    CHANGELOG.md

source-repository head
  type: git
  location: https://github.com/sol/hpack

library
  exposed-modules:
      Hpack
      Hpack.Config
      Hpack.Render
      Hpack.Yaml
  other-modules:
      Paths_hpack
  hs-source-dirs:
      src
  ghc-options: -Wall
  build-depends:
      Cabal >=2.2
    , Glob >=0.9.0
    , aeson >=1.2.1.0
    , base >=4.9 && <5
    , bifunctors
    , bytestring
    , containers
    , cryptonite
    , deepseq
    , directory
    , filepath
    , http-client
    , http-client-tls
    , http-types
    , infer-license >=0.2.0 && <0.3
    , pretty
    , scientific
    , text
    , transformers
    , unordered-containers
    , vector
    , yaml >=0.10.0
  default-language: Haskell2010

executable hpack
  main-is: Main.hs
  other-modules:
      Paths_hpack
  hs-source-dirs:
      driver
  ghc-options: -Wall
  build-depends:
      Cabal >=2.2
    , Glob >=0.9.0
    , aeson >=1.2.1.0
    , base >=4.9 && <5
    , bifunctors
    , bytestring
    , containers
    , cryptonite
    , deepseq
    , directory
    , filepath
    , hpack
    , http-client
    , http-client-tls
    , http-types
    , infer-license >=0.2.0 && <0.3
    , pretty
    , scientific
    , text
    , transformers
    , unordered-containers
    , vector
    , yaml >=0.10.0
  default-language: Haskell2010

test-suite spec
  type: exitcode-stdio-1.0
  main-is: Spec.hs
  other-modules:
      Paths_hpack
  hs-source-dirs:
      test
      src
  ghc-options: -Wall
  cpp-options: -DTEST
  build-tool-depends:
      hspec-discover:hspec-discover
  build-depends:
      Cabal >=2.2
    , Glob >=0.9.0
    , HUnit >=1.6.0.0
    , QuickCheck
    , aeson >=1.2.1.0
    , base >=4.9 && <5
    , bifunctors
    , bytestring
    , containers
    , cryptonite
    , deepseq
    , directory
    , filepath
    , hspec ==2.*
    , http-client
    , http-client-tls
    , http-types
    , infer-license >=0.2.0 && <0.3
    , interpolate
    , mockery >=0.3
    , pretty
    , scientific
    , template-haskell
    , temporary
    , text
    , transformers
    , unordered-containers
    , vector
    , yaml >=0.10.0
  default-language: Haskell2010
