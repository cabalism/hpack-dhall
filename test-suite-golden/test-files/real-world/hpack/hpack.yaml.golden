name: hpack
version: 0.31.1
synopsis: A modern format for Haskell packages
description: See README at <https://github.com/sol/hpack#readme>
category: Development
maintainer: Simon Hengel <sol@typeful.net>
github: sol/hpack
extra-source-files:
- CHANGELOG.md
ghc-options: -Wall
dependencies:
- base >= 4.9 && < 5
- bytestring
- deepseq
- directory
- filepath
- Glob >= 0.9.0
- text
- containers
- unordered-containers
- yaml >= 0.10.0
- aeson >= 1.2.1.0
- scientific
- Cabal >= 2.2
- pretty
- bifunctors
- cryptonite
- transformers
- http-types
- http-client
- http-client-tls
- vector
- infer-license >= 0.2.0 && < 0.3
library:
  exposed-modules:
  - Hpack
  - Hpack.Config
  - Hpack.Render
  - Hpack.Yaml
  source-dirs: src
executable:
  main: Main.hs
  source-dirs: driver
  dependencies:
  - hpack
tests:
  spec:
    main: Spec.hs
    source-dirs:
    - test
    - src
    cpp-options: -DTEST
    dependencies:
    - hspec == 2.*
    - QuickCheck
    - temporary
    - mockery >= 0.3
    - interpolate
    - template-haskell
    - HUnit >= 1.6.0.0
    build-tools: hspec-discover

